### PR TITLE
Add data from Fajnzylber2020SARS

### PR DESCRIPTION
**Pull request by WenfeiXiao**
I added data from the paper titled "SARS-CoV-2 viral load is associated with increased disease severity and mortality." The paper quantified SARS-CoV-2 viral load from participants with a diverse range of COVID-19 disease severities, including those requiring hospitalization, outpatients with mild disease, and individuals with resolved infections. Nasopharyngeal swabs, oropharyngeal swabs, sputum, and urine samples were collected from hospitalized participants. Data were obtained from the supplementary materials, and I have added the extraction sheet.

I'm unsure if the original .xlsx file needs to be uploaded to the repository.

### CODE REVIEW

File: data/fajnzylber2020sars/fajnzylber2020sars.yaml [hunk 1]
```diff
@@ -0,0 +1,1733 @@
+# yaml-language-server: $schema=../.schema.yaml
+title: SARS-CoV-2 viral load is associated with increased disease severity and mortality
+doi: 10.1038/s41467-020-19057-5
+description: >
+  The paper quantified SARS-CoV-2 viral load from participants with a diverse range
+  of COVID-19 disease severity, including those requiring hospitalization, outpatients
+  with mild disease, and individuals with resolved infections. Nasopharyngeal swabs,
+  oropharyngeal swabs, sputum, and urine were collected from hospitalized participants.
```
Comment (YWAN446) [inline data/fajnzylber2020sars/fajnzylber2020sars.yaml:8]: We need to mention the collection of blood samples and the plasma samples are from blood samples.

File: data/fajnzylber2020sars/fajnzylber2020sars.yaml [hunk 2]
```diff
@@ -0,0 +1,1733 @@
...
+  Nasopharyngeal_SARSCoV2_N:
+    description: >
+      Nasopharyngeal swabs were collected in 3 mL of phosphate-buffered saline. Viral
+      concentrations were quantified using RT-qPCR targeting the N gene in the nasopharyngeal
+      swab samples.
+    limit_of_quantification: unknown
+    limit_of_detection: 40
+    specimen: nasopharyngeal_swab
+    biomarker: SARS-CoV-2
+    gene_target: N
```
Comment (YWAN446) [inline data/fajnzylber2020sars/fajnzylber2020sars.yaml:20]: Should be "N1" for all the analytes.

File: data/fajnzylber2020sars/fajnzylber2020sars.yaml [hunk 3]
```diff
@@ -0,0 +1,1733 @@
...
+  The paper quantified SARS-CoV-2 viral load from participants with a diverse range
+  of COVID-19 disease severity, including those requiring hospitalization, outpatients
+  with mild disease, and individuals with resolved infections. Nasopharyngeal swabs,
+  oropharyngeal swabs, sputum, and urine were collected from hospitalized participants.
+  Data were obtained from the supplementary materials.
+analytes:
+  Nasopharyngeal_SARSCoV2_N:
+    description: >
+      Nasopharyngeal swabs were collected in 3 mL of phosphate-buffered saline. Viral
+      concentrations were quantified using RT-qPCR targeting the N gene in the nasopharyngeal
```
Comment (YWAN446) [inline data/fajnzylber2020sars/fajnzylber2020sars.yaml:14]: Should be "N1" for all the analytes.

File: data/fajnzylber2020sars/fajnzylber2020sars.yaml [hunk 4]
```diff
@@ -0,0 +1,1733 @@
...
+    limit_of_quantification: unknown
+    limit_of_detection: 40
+    specimen: oropharyngeal_swab
+    biomarker: SARS-CoV-2
+    gene_target: N
+    unit: gc/mL
+    reference_event: symptom onset
+  Oropharyngeal_VTM_SARSCoV2_N:
+    description: >
+      Oropharyngeal swabs were collected in viral transport medium. Viral concentrations
```
Comment (YWAN446) [inline data/fajnzylber2020sars/fajnzylber2020sars.yaml:37]: Do we know the volume of the viral transport medium? Is that also 3 mL?

File: data/fajnzylber2020sars/fajnzylber2020sars.yaml [hunk 5]
```diff
@@ -0,0 +1,1733 @@
...
+      RT-qPCR targeting the N gene.
+    limit_of_quantification: unknown
+    limit_of_detection: 40
+    specimen: urine
+    biomarker: SARS-CoV-2
+    gene_target: N
+    unit: gc/mL
+    reference_event: symptom onset
+participants:
+- attributes:
```
Comment (YWAN446) [inline data/fajnzylber2020sars/fajnzylber2020sars.yaml:80]: Need to create `race` and `ethnicity`: W = White; H = Hispanic; B = Black; O = Other; U = Unknown
.  Please check the scheme to see what are the current settings for `race` and `ethnicity`.

File: data/fajnzylber2020sars/fajnzylber2020sars-extraction.md [hunk 6]
```diff
@@ -0,0 +1,145 @@
+# Extraction for Natarajan et al. (2022)
```
Comment (YWAN446) [inline data/fajnzylber2020sars/fajnzylber2020sars-extraction.md:1]: This line needs to be changed to Fajnzylber et al. (2020).

File: data/fajnzylber2020sars/fajnzylber2020sars-extraction.md [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+
+[Fajnzylber et al. (2020)](https://doi.org/10.1038/s41467-020-19057-5) quantified SARS-CoV-2 viral load from participants with a diverse range of COVID-19 disease severity, including those requiring hospitalization, outpatients with mild disease,  and individuals with resolved infections. Nasopharyngeal swabs, oropharyngeal swabs, sputum, and urine were collected from hospitalized participants. Data were obtained from the supplementary materials.
```
Comment (YWAN446) [inline data/fajnzylber2020sars/fajnzylber2020sars-extraction.md:3]: It collected blood samples.

**Review by YWAN446 — COMMENTED** [review]
Please also upload the raw data "41467_2020_19057_MOESM3_ESM.xlsx".

File: data/fajnzylber2020sars/fajnzylber2020sars.yaml (hunk 5 above)
Comment (YWAN446) [reply data/fajnzylber2020sars/fajnzylber2020sars.yaml:80]: May also include the `death` as an attribute.

**Comment by YWAN446** [comment]
Do you think the data only include inpatients, that are hospitalized participates?

File: data/fajnzylber2020sars/fajnzylber2020sars.yaml (hunk 1 above)
Comment (WenfeiXiao) [reply data/fajnzylber2020sars/fajnzylber2020sars.yaml:8]: Good point!

**Comment by WenfeiXiao** [comment]
Blood samples were collected from hospitalized participants, non-hospitalized symptomatic individuals seeking care at a respiratory infection clinic, and participants who had recovered from known COVID-19. Other samples were collected only from hospitalized participants. Thanks for pointing out the confusion! I'll update this in the description.

File: data/fajnzylber2020sars/fajnzylber2020sars.yaml (hunk 4 above)
Comment (WenfeiXiao) [reply data/fajnzylber2020sars/fajnzylber2020sars.yaml:40]: The methodology section mentioned that 'Nasopharyngeal swabs and oropharyngeal swabs were collected in 3 mL of phosphate-buffered saline,' but it did not specify the volume of the viral transport medium. Therefore, I did not include that information in the analyte description.

File: data/fajnzylber2020sars/fajnzylber2020sars.yaml (hunk 5 above)
Comment (WenfeiXiao) [reply data/fajnzylber2020sars/fajnzylber2020sars.yaml:83]: Good point! I updated Race and Ethnicity based on the schema. For Race_Ethnicity = H, the race is unknown, and the ethnicity is Hispanic. There is also BMI and diabetes information that can be added to subject attributes. Should I add this information to the .yaml file and update the schema as well?

File: data/fajnzylber2020sars/fajnzylber2020sars-extraction.md [hunk 8]
```diff
@@ -0,0 +1,175 @@
+# Extraction for Fajnzylber et al. (2020)
+
+[Fajnzylber et al. (2020)](https://doi.org/10.1038/s41467-020-19057-5) quantified SARS-CoV-2 viral load from participants with a diverse range of COVID-19 disease severity, including those requiring hospitalization, outpatients with mild disease, and individuals with resolved infections. Blood was collected from hospitalized participants, non-hospitalized symptomatic individuals seeking care at a respiratory infection clinic, and participants who had recovered from known COVID-19 disease. Nasopharyngeal swabs, oropharyngeal swabs, sputum, and urine were collected from hospitalized participants. Data were obtained from the supplementary materials.
```
Comment (YWAN446) [inline data/fajnzylber2020sars/fajnzylber2020sars-extraction.md:3]: For the data in the supplementary materials. Could you add the link for it?

Comment (YWAN446) [reply data/fajnzylber2020sars/fajnzylber2020sars-extraction.md]: I added it.

**Review by YWAN446 — APPROVED** [review]

**Pull request merged**